src/base64-inl.h

```diff
@@ -182,6 +211,96 @@ inline size_t base64_encode(const char* src,
   return dlen;
 }
 
+
+#if (defined(__x86_64) || defined(__x86_64__)) && \
+    (defined(__linux) || defined(__linux__))
+#pragma GCC target("avx512vl", "avx512vbmi")
```

Comment:
Would it be a bit cleaner to use `__attribute__((target(...)))` on the function itself? Or is this necessary for the #include to work?

Reply:
I chose #pragma so to explicitly reset the pragma to avoid polution on other code.